### PR TITLE
MEN-3519: filter by Device ID

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: src/js/components/devices/filteritem.js [hunk 1]
```diff
@@ -143,7 +149,7 @@ export default class FilterItem extends React.Component {
           options={filters.sort((a, b) => a.priority - b.priority)}
           renderInput={params => <TextField {...params} label="Attribute" style={textFieldStyle} />}
           key={reset}
-          value={key}
+          value={label}
```
Comment (mzedel) [inline src/js/components/devices/filteritem.js:152]: with the state stored label this leads to stale values in the autocomplete (e.g. if you clear the value) - doing it on render as you do in the [filters](https://github.com/mendersoftware/gui/pull/759/files#diff-1b8fef05bf9dba7c806bcefec3265fc7R145) should be enough here as well

Comment (tranchitella) [reply src/js/components/devices/filteritem.js:152]: Done!

File: src/js/components/devices/filteritem.js [hunk 2]
```diff
@@ -62,13 +63,18 @@ export default class FilterItem extends React.Component {
     return !(nextState.key === this.state.key && nextState.value === this.state.value && nextState.operator === this.state.operator);
   }
 
+  getFilterLabelByKey(key) {
```
[the remark this answers is not in the document]
Comment (tranchitella) [reply src/js/components/devices/filteritem.js:66]: Done!

**Comment by tranchitella** [comment]
@mzedel I've updated the PR with the changes you requested. I don't dare to change and merge getGroupDevices and getDevicesByStatus. Do you mind if I skip this part?

**Pull request merged**